hook!-full-game/src/UI/debug/DebugPanel.gd

```diff
@@ -0,0 +1,38 @@
+extends Control
+
+onready var reference_node: = get_node(reference_node_path)
+onready var container: VBoxContainer = $MarginContainer/VBoxContainer
+
+export (NodePath) var reference_node_path: NodePath = "."
+export (PoolStringArray) var properties = []
+
+func _ready() -> void:
+	setup_properties()
+
+
+func _process(delta) -> void:
```

Comment:
I couldn't get the labels to update. They only display the starting value.

Reply:
Can you check the example and see if you can manage to make them work? It is strange, tho. If they display they should update. Anyway with the demo it might be clearer.